### PR TITLE
(PE-23817) Update OpenSSL for pe-installer-runtime

### PR DESCRIPTION
**Pull request by highb**

### CODE REVIEW

File: configs/projects/pe-installer-runtime.rb [hunk 1]
```diff
@@ -95,7 +96,7 @@
     'no-dtls1',
     'no-idea',
     'no-seed',
-    'no-ssl2-method',
+    # 'no-ssl2-method',
```
Comment (branan) [inline configs/projects/pe-installer-runtime.rb]: should this be removed entirely?

Comment (highb) [reply configs/projects/pe-installer-runtime.rb]: Maybe? I was just copy-pastaing from the pdk-runtime. 🤷‍♂️

**Comment by branan** [comment]
Do you need to branch the runtime for pre-johnson and johnson separately, or are you comfortable with older installers ending up with the updated openssl?

**Comment by highb** [comment]
I'm not even sure we'll start using this runtime in Johnson, and I'm pretty sure we won't be back porting it.

**Comment by highb** [comment]
@branan Removed the commented out openssl option.

**Comment by branan** [comment]
:+1: wasn't sure where/how it was being used. If it's still all in-development it doesn't matter to me :)

**Pull request merged**